pkg/discovery/dtofactory/service_entity_dto_builder.go

```diff
@@ -264,7 +265,7 @@ func getIPProperty(pods []*api.Pod) *proto.EntityDTO_EntityProperty {
 	attr := stitching.AppStitchingAttr
 	ips := []string{}
 	for _, pod := range pods {
-		ips = append(ips, servicePrefix+"-"+pod.Status.PodIP)
+		ips = append(ips, servicePrefix+"-"+pod.Status.PodIP+"-"+ParsePodUID(string(pod.UID)))
```

Comment:
This is not right. We want to append the Kubernetes service ID (see [this](https://github.com/turbonomic/kubeturbo/blob/fae80794057055fadfff21a8879c12530a590e53/pkg/cluster/cluster_info_scraper.go#L327)), not the pod ID. The `prometurbo` probe cannot reliably discover pod ID, but it can discover Kubernete service ID.